src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -145,6 +148,32 @@ def on_time_step_prepare(self, event: Event):
         propensity.loc[remitted_mask] = self.randomness.get_draw(remitted_mask.index)
         self.population_view.update(propensity)
 
+    ##################################
+    # Pipeline sources and modifiers #
+    ##################################
+
+    def _get_current_exposure(self, index: pd.Index) -> pd.Series:
+        is_mam_component = self.risk.name == "moderate_acute_malnutrition_treatment"
+        coverage_to_exposure_map = {"none": "cat1", "full": "cat2"}
+
+        if is_mam_component:
```

Comment:
The if/else seems to duplicate some code here; you should be able to avoid this with something like
`coverage = self.scenario.mam_tx_coverage if self.risk.name =="moderate_acute_malnutrition_treatment" else self.scenario.sam_tx_coverage`
or a dictionary would be fine too

Reply:
That would be better and I thought about how to limit duplication but it was getting late yesterday and I knew this worked haha. I'll keep it for now so we don't run into any unexpected consequences.